.gitignore

```diff
@@ -60,3 +60,5 @@ cmake-build-*/
 .DS_Store
 .AppleDouble
 .LSOverride
+
+*.png
```

Comment:
Probably should not add this because we already have a lot of png files in the repo.

Reply:
That was unintentional, sorry